cypress/e2e/AddingEmployees.ts

```diff
@@ -0,0 +1,59 @@
+import AddEmployeePage from "../pages/AddEmployeePage";
+import DashBoardPage from "../pages/DashBoardPage";
+import LoginPage from "../pages/LoginPage";
+import PIMPage from "../pages/PIMPage";
+import EmployeeListPage from "../pages/EmployeeListPage";
+import PersonalDetailsPage from "../pages/PersonalDetailsPage";
+
+describe("Adding New Employee Functionality", () => {
+  describe("Add New Employee With Admin User", () => {
+    beforeEach(() => {
+      LoginPage.navigateTo(LoginPage.path);
+      LoginPage.login(Cypress.env("username"), Cypress.env("password"));
+    });
+
+    it("As admin I should add a new employee succussfuly", () => {
+      LoginPage.navigateTo(DashBoardPage.path);
```

Comment:
you meant 
DashBoardPage.navigateTo(DashBoardPage.path);

Reply:
I am in the LoginPage, and the I will navigate to the DashBoardPage